main.cpp

```diff
@@ -63,12 +66,49 @@ send me a DM to check your pull request
 //2)
 void variableDeclarations()
 {
+    
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
+
+    int beer = 4;
+    int bread = 1;
+    int meat = 3;
+
+    char ball = 1;
+    char vol = 2;
+    char wheel = 4;
+
+    bool rolling = false;
+    bool sync = true;
+    bool start = false;
+
+    float frequence = 404.457676f;
+    float resonance = 123.78987f;
+    float modulation = 89.7484f;
+
+    double chorus = 34.9484884;
+    double flanger = 89.939090;
+    double delayTime = 3.989898989;
     
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    ignoreUnused(beer);
+    ignoreUnused(bread);
+    ignoreUnused(meat);
+    ignoreUnused(ball);
+    ignoreUnused(vol);
+    ignoreUnused(wheel);
+    ignoreUnused(rolling);
+    ignoreUnused(sync);
+    ignoreUnused(start);
+    ignoreUnused(frequence);
+    ignoreUnused(resonance);
+    ignoreUnused(modulation);
+    ignoreUnused(chorus);
+    ignoreUnused(flanger);
+    ignoreUnused(delayTime);
```

Comment:
- [ ] Pass them all to a single call of `ignoreUnused`, like you do in the functions below.

Reply:
ok. done.